ui/src/utils/modelImageProcessor.js

```diff
@@ -0,0 +1,9 @@
+/**
+ *
+ * @param {Array} files - File hits from GraphQL query
```

Comment:
would rather add in flowtype over javadoc style type annotations; then could get flow check errors. What do you think?

Reply:
Yeah or maybe even both? Flow can enforce the types but I find the comments help future developers (or even future self) when updating or refactoring. We should make a new ticket for flow types then.